Procfile

```diff
@@ -1,4 +1,4 @@
-web: gunicorn opencraft.wsgi --log-file -
+web: gunicorn opencraft.wsgi --timeout 60 -w 4 --log-file -
```

Comment:
Please use the long form of the flag here.  This way, it's immediately obvious what all the options do, and it's also consistent with the other flags in this file.